### PR TITLE
Created Main Page and Define Routes for Other Pages

### PR DESCRIPTION
**Pull request by samclusker**

### CODE REVIEW

File: bondible-app/src/pages/MainPage.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    img: {
+      position: 'relative',
+      top: '200px',
+      maxWidth: '100%',
+      height: 'auto'
+    }
+  });
+
+  const MainPage: React.FC = (): any => {
```
Comment (harrymoy) [inline bondible-app/src/pages/MainPage.tsx]: Is there a specific reason to use React.FC and declare the return type here?

**Review by harrymoy — COMMENTED** [review]
All looks great, just curious about the React.FC stuff.

File: bondible-app/src/pages/MainPage.tsx (hunk 1 above)
Comment (samclusker) [reply bondible-app/src/pages/MainPage.tsx]: Removed type any but reckon declaring as Functional Component is worth its time?

Comment (harrymoy) [reply bondible-app/src/pages/MainPage.tsx]: I'm reading online that React.FC isn't necessary, but it's also not deprecated. We're using it in some places but not in others 🤣 so we should decide whether we use it or not for the sake of consistency.

Comment (samclusker) [reply bondible-app/src/pages/MainPage.tsx]: Lets sack it off then, we know its an FC. Was only declaring it because TYPESCRIPT

**Pull request merged**